model/postgres/postgres_server.rb

```diff
@@ -83,10 +83,16 @@ def configure_hash
         configs[:primary_conninfo] = "'#{resource.replication_connection_string(application_name: ubid)}'"
       end
 
-      if doing_pitr?
+      if doing_pitr? && !read_replica?
         configs[:recovery_target_time] = "'#{resource.restore_target}'"
       end
 
+      if read_replica?
+        configs[:recovery_target_time] = "''"
+        configs[:recovery_target_timeline] = "latest"
+        configs[:recovery_target_inclusive] = false
+      end
+
```

Comment:
nit: It might be personal, but it feels easier to read for me
```suggestion
      if read_replica?
        configs[:recovery_target_time] = "''"
        configs[:recovery_target_timeline] = "latest"
        configs[:recovery_target_inclusive] = false
      elsif doing_pitr?
        configs[:recovery_target_time] = "'#{resource.restore_target}'"
      end
```